docker/docker-compose.yml

```diff
@@ -133,11 +133,11 @@ services:
 
   minio:
     <<: *project_defaults
-    image: minio/minio
+    image: minio/minio:${MINIO_IMAGE-RELEASE.2023-08-23T10-07-06Z}
```

Comment:
I think it is better to set the variable to the entire image name not just the version : 
`MINIO_IMAGE=minio/minio:RELEASE.2023-08-23T10-07-06Z`

So that users could change easily the image if they want.